src/api/searchApi.ts

```diff
@@ -19,7 +19,18 @@ export const fetchTickets = async (results: number, from: number, query = '', on
 
 export const fetchImportCandidates = async () => {
   const getImportCandidates = await authenticatedApiRequest2<SearchResponse<ImportCandidateSummary>>({
-    url: `${SearchApiPath.ImportCandidates}`,
+    url: SearchApiPath.ImportCandidates,
   });
   return getImportCandidates.data;
 };
+
+export const fetchRegistrationsExport = async (queryParam: string) => {
+  let url: string = SearchApiPath.RegistrationsExport;
+  if (queryParam) {
+    url += `?query=${queryParam}`;
+  }
+
+  // TODO: Set 'Accept: text/csv' in request header when supported by the API (NP-44982)
```

Comment:
Kan avvente bittelitt for avklaring her: https://sikt-no.slack.com/archives/C012AQNU960/p1688215474531339